src/sxastarter/package.json

```diff
@@ -35,8 +35,8 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "@sitecore-feaas/clientside": "^0.3.10",
-    "@sitecore-jss/sitecore-jss-nextjs": "~21.2.4",
+    "@sitecore-feaas/clientside": "0.3.10",
```

Comment:
I don't think we want to lock it here (keep caret) as this would be a diff from our template, just keep the lock file unchanged (set at 0.3.10).

Reply:
Ken asked to lock it a couple comments up
I see value in either approach, which one should we go for?

Reply:
I read the comment as just keeping the version in the lock file at 0.3.10 (and keeping our package.json dependency unchanged - using "^0.3.10"). I'd recommend this approach. @kendoce thoughts?